### PR TITLE
feat: make avatar a shiba, add slides

### LINKED ISSUE #5: 🚀 Feature: Make better avatars

### LINKED ISSUE #25: 🐛 Bug: rotate avatar
Avatar is turned away from the stage on spawn

--- 

This is a :bug: Bug Report

### PR DESCRIPTION
**Pull request by DonLe27**
It's a shiba

Resolves #5

### CODE REVIEW

**Comment by DonLe27** [comment]
TODO: the face disappears if you look at the shiba from a certain angle. I'll fix it after Thursday but if you have any ideas please comment lmao

File: server/src/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const http = require('http');
+const dotenv = require('dotenv');
```
Comment (BryanPan342) [inline server/src/index.js]: remove this change

Comment (BryanPan342) [inline server/src/index.js]: remove this change

File: client/src/assets/steve.obj [hunk 2]
```diff
@@ -0,0 +1,308 @@
+# Blender v2.74 (sub 0) OBJ File: ''
```
Comment (BryanPan342) [inline client/src/assets/steve.obj]: remove this file

**Review by BryanPan342 — CHANGES_REQUESTED** [review]
remove changes:
- server/src/index.js

remove files:
- **/steve.*
- assets/test.glb

only changes i need to hit merge

**Review by BryanPan342 — APPROVED** [review]
LGTM

**Pull request merged**